src/lib/proxy-server.ts

```diff
@@ -97,14 +97,16 @@ export async function startProxyServer(): Promise< boolean > {
 		return true;
 	} catch ( error ) {
 		if ( error instanceof Error && 'code' in error && error.code === 'EADDRINUSE' ) {
-			dialog.showMessageBox( {
-				type: 'error',
-				message: __( 'Failed to start custom domain proxy server' ),
-				detail: __(
-					'Another server is already running on port 80. For custom domains to work, please stop that server and then restart Studio.'
-				),
-				buttons: [ __( 'OK' ) ],
-			} );
+			setTimeout( () => {
+				dialog.showMessageBox( {
+					type: 'error',
+					message: __( 'Custom domain set up failed' ),
+					detail: __(
+						'Studio needs to use port 80, but it’s in use by another app. Close any local development apps and restart Studio.'
```

Comment:
```suggestion
						'Studio needs to use port 80, but it’s already in use by another app. Close any local development apps and restart Studio.'
```
Not a native English speaker, but this sounds better, IMO.

Reply:
That sounds good to me. 👍🏼